python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -54,19 +55,57 @@ def transpile(self, program, place, scope=None):
         if not isinstance(scope, core.Scope):
             raise TypeError("scope should be as Scope type or None")
         self.fuse_batch_norm(program, place, scope)
+        use_mkldnn = bool(os.getenv("FLAGS_use_mkldnn", False))
+        if use_mkldnn:
+            self.fuse_relu(program)
+
+    def fuse_relu(self, program):
```

Comment:
- how about the function name of `fuse_relu_mkldnn`?
- you may add the check of `FLAGS_use_mkldnn=True` in this function, thus, other people will not use it in plain CPU.